app/main.py

```
@@ -387,6 +388,7 @@ async def validation_exception_handler(request: Request, exc: RequestValidationE
         )
 
     app.include_router(roll.router, prefix="/api/roll", tags=["roll"])
+    app.include_router(review.router, prefix="/api/reviews", tags=["reviews"])
```

Ruff (0.15.9)

[error] Undefined name `review`

(F821)